import.py

```diff
@@ -40,13 +41,41 @@ def main():
 
   first = True
   for f in os.listdir(dataDir):
-    if f[-4:].lower() == ".shp":
+    rasterFound = False
+    shapefileFound = False
+    if f[-4:].lower() == ".tif":
+      rasterFound = True
+      stem = f[:-4].replace(".", "_").replace("-","_")
+      print("Opening raster data source:", stem)
+      rst = processRaster(f, stem, dataDir, reprojectedDir, SRIDNamespace, desiredSRID)
+      keyField = "lookup_val"
+      shapeType = "Raster"
+      # TODO: figure out if we need to reproject this to EPSG:4326 or can just run with whatever we're given
```

Comment:
looks like we need to reproject it, or am I misunderstanding a conversation we had the other day?

Reply:
You are correct.  I just haven't cleaned up the todo comment yet....